server.js

```diff
@@ -21,8 +21,8 @@
 const http = require('http');
 const { MongoClient } = require('mongodb');

```

Comment:
Consider using a descriptive name instead of 'HOSTNAME'. For example, 'SERVER_HOST' might be more informative.